catalog/security/catalog-security-filter/src/main/java/ddf/catalog/security/filter/plugin/FilterPlugin.java

```diff
@@ -100,16 +107,18 @@ public QueryResponse process(QueryResponse input)
             }
         }
 
-        logger.info("Filtered {} metacards, returned {}", filteredMetacards,
+        logger.info("Filtered {} metacards, returned {}",
+                filteredMetacards,
                 (newResults.size() - filteredMetacards));
```

Comment:
Looks like the logger and SecurityLogger will return the wrong number of metacards returned.